src/components/Admin/EntryCard.vue

```diff
@@ -215,11 +216,18 @@ function onPaste(evt) {
 }
 
 async function onSubmit() {
+  entry.title = entry.title.trim()
   const hasEntry = await entryStore.hasEntry(entry.prompt?.value)
   if (!props.id && hasEntry) {
     $q.notify({ type: 'info', message: 'You have already submitted an entry for this prompt. Please select another prompt' })
     return
   }
+
+  const entryNameValidator = await entryStore.entryNameValidator(props.id, entry.prompt?.value, entry.title, !!props.id)
```

Comment:
Is there a way we could first check if there are loaded entries already? Maybe scrolled down the prompt before creating an entry, and we already have the list of entries and there is no need then to fetch again for all the entries of the prompt. 🤔

Reply:
I will look into it.

Reply:
I have updated this logic.